state/migration_export.go

```diff
@@ -1176,15 +1176,10 @@ func (e *exporter) subnets() error {
 			ProviderNetworkId: string(subnet.ProviderNetworkId()),
 			VLANTag:           subnet.VLANTag(),
 			SpaceName:         subnet.SpaceName(),
+			AvailabilityZones: subnet.AvailabilityZones(),
```

Comment:
To answer @jameinel question from prior pr: So for migrations, have they always supported zones as a list and we just only ever transferred a single one? (so no compatibility issues with changing the stream?)

Yes.